src/main/java/com/cs3365/rpgdapi/weapons/WeaponsController.java

```diff
@@ -64,4 +59,40 @@ public ResponseEntity get(@PathVariable("weaponID") UUID identifier)
                                          e.getMessage()));
         }
     }
+
+    @PutMapping("/{id}")
+    public ResponseEntity<String> put(@RequestBody Weapon requestBody,
```

Comment:
So I changed it, but I ended up getting rid of it, because that first try catch block with `.validate()` couldn't return the `.badRequest` message due to incompatible types. I don't know if there's another way to do this.